### PR TITLE
Add Voz.vn Rich Presence

### PR DESCRIPTION
**Pull request by TheDyet94**

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy errors.

File: websites/V/Voz.vn/presence.ts [hunk 1]
```diff
@@ -0,0 +1,175 @@
+const presence = new Presence({
+  clientId: "773624839268597820"
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: presenceData = {
```
Comment (ririxichan) [inline websites/V/Voz.vn/presence.ts:8]: this is PresenceData

File: websites/V/Voz.vn/presence.ts [hunk 2]
```diff
@@ -0,0 +1,175 @@
+const presence = new Presence({
+  clientId: "773624839268597820"
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (ririxichan) [inline websites/V/Voz.vn/presence.ts:5]: merge consts

File: websites/V/Voz.vn/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "name": "U+0110",
+    "id": "773624839268597820"
+  },
+  "service": "Voz.vn",
+  "description": {
+    "en": "Vietnam surrenders to forum technology :|",
+    "vi_VN": "Việt Nam đầu hàng công nghệ diễn đàn :|"
+  },
+  "url": "voz.vn",
+  "version": "1.0.1",
```
Comment (ririxichan) [inline websites/V/Voz.vn/dist/metadata.json:12]: 1.0.0

**Review by ririxichan — CHANGES_REQUESTED** [review]

**Comment by ririxichan** [comment]
No response from author. If you want to do pull requested with fixed changes, sure. For now I am closing this pull request.

**Pull request closed without merging**